addons/html_editor/static/src/core/history_plugin.js

```diff
@@ -660,6 +666,7 @@ export class HistoryPlugin extends Plugin {
         }
         // Reapply the uncommited draft, since this is not an operation which should cancel it
         this.applyMutations(this.currentStep.mutations);
+        this.dispatch("NORMALIZE", { node: this.editable });
```

Comment:
@Mtaylorr 
Now that I think about it, we should put this normalize right after
```js
this.applyMutations(newStep.mutations);
this.dispatch("NORMALIZE", { node: this.getMutationRoot(newStep.mutations) || this.editable });
```

In order to to the minimal amount of work, at the right time (since only received nodes may not be normalized)